tests/Doctrine/Tests/Common/Persistence/ManagerRegistryTest.php

```diff
@@ -64,12 +63,12 @@ public function testGetManagerForInvalidAliasedClass()
     }
 }
 
-class TestManager
+class TestManager extends DoctrineTestCase
```

Comment:
> Can't we just extend a PHPUnit_Framework_TestCase?

You’re right, PR updated, thanks.

Please note that the tests still fail when calling the file alone (within the global phpunit call it works fine):
```sh
$ phpunit tests/Doctrine/Tests/Common/Persistence/ManagerRegistryTest.php
[…]
There were 2 errors:

1) Doctrine\Tests\Common\Persistence\ManagerRegistryTest::testGetManagerForClass
ReflectionException: Class Doctrine\Tests\Common\Persistence\TestObject does not exist

/home/taffit/debian/php-doctrine-common/lib/Doctrine/Common/Persistence/AbstractManagerRegistry.php:197
/home/taffit/debian/php-doctrine-common/tests/Doctrine/Tests/Common/Persistence/ManagerRegistryTest.php:38
```